README.md

```diff
@@ -30,7 +30,7 @@ your answer to this markdown file.
 
 ---------------------------------------------------------
 
-An invariant of this fibonacci sequence could be, using a base case of [0,1], an array of [0,1,i] where i = 0 + 1.  So assuming that the function is doing the operations correctly the full invariant would be an array up to the chosen index where i is equal to the sum of the two previous entries, so for the provided index = 7 the invariant for this position would be [0,1,1,2,3,5,8].  
+An invariant of this fibonacci sequence could be, using a base case of [0,1], an array of [0,1] entering the 3rd recursion, so the invariant should be the array with the values leading up to but not including the current iteration.  So assuming that the function is doing the operations correctly the full invariant would be an array up to the chosen index where i is equal to the sum of the two previous entries, so for the provided index = 7 the invariant for this position would be [0,1,1,2,3,5,8].  
```

Comment:
Don't know what you mean by this. What elements does the array contain?

Reply:
The array contains all the values of the Fibonacci sequence up to the desired number, so for example for fib(7) then the invariant array is [0,1,1,2,3,5,8] but not [0,1,1,2,3,5,8,13]

Reply:
Then please say that in the invariant.